samtranslator/model/api/http_api_generator.py

```diff
@@ -489,14 +489,21 @@ def _set_default_authorizer(self, open_api_editor, authorizers, default_authoriz
                 path, default_authorizer, authorizers=authorizers, api_authorizers=api_authorizers
             )
 
-    def _get_authorizers(self, authorizers_config, default_authorizer=None):
+    def _get_authorizers(self, authorizers_config, enable_iam_authorizer=False):
```

Comment:
Just making sure I understand here. It seems like `default_authorizer` was never used in this function so it should be safe to updated this. I just want to double check what I am seeing/understanding to make sure there is no accidentally side affect here.

Reply:
yea. I was a bit surprised when I noticed it appeared to be unused. As far as I can tell it was unused.

Reply:
Used in `ApiGenerator` so maybe it was copied from there.